frontend/src/components/UserMailboxAuthorizations.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <v-card  v-if="clientsWithRoles.length > 0" outlined class="subgroup">
+    <h2 class="sfds-authorizations-header">Mailbox Authorizations</h2>
+
+    <div class="nav-tabs" style="width: 100%">
+    <!-- // todo: ??????????? why does this NOT work if its selectedClient = 1?? -->
```

Comment:
Is this comment still applicable?

Reply:
no - I'll delete all these comments, thanks